src/containers/shared/components/Transaction/XChainAddAttestation/types.ts

```diff
@@ -0,0 +1,90 @@
+export interface IssuedCurrency {
+  currency: string
+  issuer: string
+}
+
+export interface IssuedCurrencyAmount extends IssuedCurrency {
+  value: string
+}
+
+export type Amount = IssuedCurrencyAmount | string
+
+export interface XChainBridge {
+  LockingChainDoor: string
+  LockingChainIssue: 'XRP' | IssuedCurrency
+  IssuingChainDoor: string
+  IssuingChainIssue: 'XRP' | IssuedCurrency
+}
+
+export interface XChainAddAttestation {
+  TransactionType: 'XChainAddAttestation'
+
+  XChainAttestationBatch: {
+    XChainBridge: XChainBridge
+
+    XChainClaimAttestationBatch: Array<{
+      XChainClaimAttestationBatchElement: {
+        Account: string
+
+        Amount: Amount
+
+        AttestationRewardAccount: string
+
+        Destination: string
+
+        PublicKey: string
+
+        Signature: string
+
+        WasLockingChainSend: 0 | 1
```

Comment:
Might be nice to have a quick comment here highlighting what a 1 means

Reply:
It's just a boolean